apps/nextjs-website/src/components/PressReleaseList.tsx

```diff
@@ -6,6 +6,23 @@ import { LocalizeURL } from '@/lib/linkLocalization';
 import { PressReleaseList as PressReleaseListRC } from '@react-components/components';
 import { PressReleaseListProps } from '@react-components/types';
 
+const formatDateToLocale = (
+  dateString: string,
+  locale: string = 'it'
```

Comment:
```suggestion
  locale: string,
```
No need for a default value since the parameter is required.